spk/java-11-openjdk/Makefile

```diff
@@ -7,7 +7,7 @@ SPK_VERS = $(JAVA_VERSION).$(JAVA_RELEASE)
 SPK_REV = 1
 SPK_ICON = src/$(SPK_NAME).png
 
-DEPENDS  = cross/$(SPK_NAME)
+DEPENDS  = cross/$(SPK_NAME) cross/zlib cross/libpng
```

Comment:
please add additional dependencies to cross/java-11-openjdk/Makefile

Reply:
Done. The version I have just pushed runs on my DS418 :)